tests/foreman/destructive/test_leapp_satellite.py

```diff
@@ -49,3 +49,6 @@ def test_positive_leapp(target_sat):
     result = target_sat.execute('cat /etc/redhat-release | grep -Po "\\d"')
     # Check if RHEL was upgraded
     assert result.stdout[0] == str(orig_rhel_ver + 1), 'RHEL was not upgraded'
+    # Check satellite's health
+    sat_health = target_sat.execute('satellite-maintain health check')
+    assert sat_health.status == 0, 'Satellite health check failed'
```

Comment:
What is the significance of `assert sat_health.status == 0, 'Satellite health check failed'` ?

Reply:
The test did not check if the satellite is still present after upgrading.

Reply:
But is the sat actually upgraded `str(orig_rhel_ver + 1), 'RHEL was not upgraded'`?